Src/Base/AMReX_FabArray.H

```diff
@@ -817,6 +817,10 @@ public:
     template <class F=FAB, std::enable_if_t<IsBaseFab<F>::value,int> = 0>
     void setDomainBndry (value_type val, int strt_comp, int ncomp, const Geometry& geom);
 
+    // Get capacity of the FabArray
+    template <class F=FAB, std::enable_if_t<IsBaseFab<F>::value,int> = 0>
```

Comment:
For flexibility, we could also make the integer type a template typename here and for in particle containers. For example,
```
template <typename I, class F=FAB, std::enable_if_t<IsBaseFab<F>::value && std::is_integral_v<I> && sizeof(I) >= sizeof(Long), int> = 0>
void capacityOfFabs (LayoutData<I>& mem) const;
```
And in the implementation, we also want to do explicit static_cast before adding the values.